BasicSample/Assets/Shared/Scripts/MixedRealityProjectSelectionWindow.cs

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.OpenXR.Remoting;
+using System;
+using System.IO;
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+using UnityEditor.XR.Management;
+using UnityEditor.XR.OpenXR.Features;
+using UnityEngine;
+using UnityEngine.XR.Management;
+using UnityEngine.XR.OpenXR;
+using UnityEngine.XR.OpenXR.Features;
+
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+    enum MixedRealityProjectConfiguration
+    {
+        None,
+        RunNativelyonPCVR,
+        RunNativelyonHL2,
+        RunRemotelyonUWP,
+        RunRemotelyonWin32
+    }
+    public class MixedRealityProjectSelectionWindow : EditorWindow
+    {
+        private MixedRealityProjectConfiguration m_selectedMRConfiguration;
+        private bool m_disablePopup;
+        private const float Default_Window_Height = 300.0f;
+        private const float Default_Window_Width = 400.0f;
+
+        public static MixedRealityProjectSelectionWindow Instance { get; private set; }
+        public static bool IsOpen => Instance != null;
+        private static bool ShowfromMenu = false;
+        private static PopupUserSettings UserSettings;
+        private const string SettingsFileName = "MixedRealityOpenXRProjectSelectionSettings.asset";
+        private static string UserSettingsFolder => Path.Combine(Application.dataPath, "..", "UserSettings");
+        private static string SettingsAssetPath => Path.Combine(UserSettingsFolder, SettingsFileName);
+
+        [MenuItem("Mixed Reality/Quick Setup", false, 499)]
+        private static void ShowWindowFromMenu()
+        {
+            ShowfromMenu = true;
+            ShowWindow();
```

Comment:
nit: I got a bit confused checking whether or not this gets reset properly (I guess it does if you restart the editor, which is the only scenario it'd need reset)

It might be more clear to have ShowWindow take showFromMenu as an argument or optional argument, instead of keeping the state on this object.

Reply:
sure, this is the code I got from somewhere else and put less thought to. Sounds good to me